testsuite/performance/tests/src/test/scala/keycloak/AdminConsoleSimulation.scala

```diff
@@ -9,33 +10,26 @@ import SimulationsHelper._
 
 
 /**
-  * @author <a href="mailto:mstrukel@redhat.com">Marko Strukelj</a>
-  */
+ * @author <a href="mailto:mstrukel@redhat.com">Marko Strukelj</a>
+ */
 class AdminConsoleSimulation extends Simulation {
 
   println()
   println("Using server: " + TestConfig.serverUrisList.get(0))
   println()
-  println("Using test parameters:")
-  println("  runUsers: " + TestConfig.runUsers)
-  println("  numOfIterations: " + TestConfig.numOfIterations)
-  println("  rampUpPeriod: " + TestConfig.rampUpPeriod)
-  println("  userThinkTime: " + TestConfig.userThinkTime)
-  //println("  badLoginAttempts: " + TestConfig.badLoginAttempts)
-  //println("  refreshTokenCount: " + TestConfig.refreshTokenCount)
-  //println("  refreshTokenPeriod: " + TestConfig.refreshTokenPeriod)
+  println("Using test parameters:\n" + TestConfig.toStringCommonTestParameters);
   println()
   println("Using dataset properties:\n" + TestConfig.toStringDatasetProperties)
 
 
   val httpProtocol = http
-    .baseURL("http://localhost:8080")
-    .disableFollowRedirect
-    .inferHtmlResources()
-    .acceptHeader("application/json, text/plain, */*")
-    .acceptEncodingHeader("gzip, deflate")
-    .acceptLanguageHeader("en-US,en;q=0.5")
-    .userAgentHeader("Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:54.0) Gecko/20100101 Firefox/54.0")
+  .baseURL("http://localhost:8080")
```

Comment:
These whitespace changes are both unrelated and lead to worse readability. Please revert in all affected files.

Reply:
Sorry for this, IDE autoformatting with different checkstyle rules. Will revert. (As for readability, that might be subjective..)